File: JavaScript/1_Condicionales/Ejercicio_1.js
/*
1.Desarrolle un algoritmo que permita leer 3 números enteros.
El algoritmo debe imprimir cual es el numero mayor y cual es el numero menor entre los 3.
Ademas ninguno de los 3 numeros ingresados pueden ser iguales.
En caso de que el usuario ingrese 2 numeros iguales debe mostrar una alerta.
*/


// Validadcion de datos ingresasados: Numeros enteros, mayores que cero, y diferentes entre si
let num1 = parseFloat(prompt('Digite un número entero y que sea mayor que cero:'));
if (!Number.isInteger(num1) || num1 <= 0){
    alert('El número no es entero o es menor que cero, ingrese un número valido')
}else{
    let num2 = parseFloat(prompt('Digite otro número entero y que sea mayor que cero:'));
    if(!Number.isInteger(num2) || num2 <= 0 || num1==num2){
        alert('Ingrese un numero valido: que sea entero, mayor que cero y diferente al los numeros ingresados')
    }else{
        let num3 = parseFloat(prompt('Ingrese un número entero, mayor que cero y diferente a los números anteriores: '));
        if(!Number.isInteger(num3) || num3 <= 0 || num3==num1 || num3==num2){
            alert('Ingrese un numero valido: que sea entero, mayor que cero y diferente al los numeros ingresados')
        }else{

            //Validando cual es el mayor
            if(num1 > num2 && num1 > num3){
                console.log('El numero mayor es: ' + num1);    
                        }
            if(num2 > num1 && num2 > num3){
                console.log('El numero mayor es: ' + num2);    
                        } 
            if(num3 > num1 && num3 > num2){
                console.log('El numero mayor es: ' + num3);    
                        }     
                        
            //Validando cual es el menor
            if(num1 < num2 && num1 < num3){
                console.log('El numero menor es: ' + num1);    
                        }
            if(num2 < num1 && num2 < num3){
                console.log('El numero menor es: ' + num2);    
                        } 
            if(num3 < num1 && num3 < num2){
                console.log('El numero menor es: ' + num3);    
                        } 
            let numeroPrueba = num1;                           
        } 
    }

}

File: Python/1_Condicionales/Ejercicio_1.py

#1.Desarrolle un algoritmo que permita leer 3 números enteros.
# El algoritmo debe imprimir cual es el numero mayor y cual es el numero menor entre los 3.
# Ademas ninguno de los 3 numeros ingresados pueden ser iguales.
# En caso de que el usuario ingrese 2 numeros iguales debe mostrar una alerta.

#Validacion de los datos
numero1 =  input('Digite un número entero: ')
numero2 = input('Ingrese otro numero: ')
if numero1 == numero2:
    print('Error: Los números son iguales, intente de nuevo ')
else:
    numero3= input('Ingrese el último número: ') 
    if numero3 == numero1 or numero3 == numero2:
        print('Error: Los números son iguales, intente de nuevo')
    else:
        if numero1 > numero2 and numero1 > numero3:
                print('Mayor = ' + numero1)
        if numero2 > numero1 and numero2 > numero3:
                print('Mayor = ' + numero2)  
        if numero3 > numero2 and numero3 > numero2:
                print('Mayor = ' + numero3)

        if numero1 < numero2 and numero1 < numero3:
                print('Menor = ' + numero1)
        if numero2 < numero1 and numero2 < numero3:
                print('Menor = ' + numero2)  
        if numero3 < numero2 and numero3 < numero2:
                print('Menor = ' + numero3)                    
            





      